src/lando/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import call_command
+
+import pytest
+
+
+@pytest.fixture
+def lando_version():
+    # The version file may or may not exist in the testing environment.
+    # We'll explicitly generate it to ensure it's there.
+    call_command('generate_version_file')

```

Comment:
Single quotes here, looks like this isn't being checked in the styling tests. I wonder if we should just run black/ruff on everything and forget about maintaining a list of lint paths.

Reply:
`lando.ui` will have many lint fixes if we do that. I'm on board with doing that in a separate PR and dealing with any issues that come about.

Reply:
I'll add the `tests` dir to the `LINT_PATHS` for now.